src/vs/languages/css/common/parser/cssScanner.ts

```diff
@@ -43,12 +43,13 @@ export enum TokenType {
 	Resolution,
 	Comma,
 	Charset,
-
 	EscapedJavaScript,
 	BadEscapedJavaScript,
 	Comment,
 	SingleLineComment,
 	EOF,
+	Var,
```

Comment:
we shouldn't have a special token for `var`. `var` should stay an identifier as there might be selectors of that name, or many other places where `var` is a valid identifier.